docs/source/gettingstarted.rst

```diff
@@ -3,15 +3,53 @@
 Installing Mantarray File Manager
 ==================================
 
-This section will demonstrate how to install the package and get started with it.
+We recommend using Jupyter Notebooks to install the mantarray-file-manager. This
+section will walk you through the process of using Jupyter notebooks and installing the
+package.
 
-Download mantarray_file_manager
---------------------------------
 
- * If you have already installed PIP and Python on your device, then run the command "pip install mantarray_file_manager"
- * If you haven't installed Python proceed to the following URL: https://www.python.org/downloads/
- * If you haven't installed PIP run in your command line (WINDOWS): "python get-pip.py"
- * If you haven't installed PIP run in your terminal (MAC): "sudo easy_install pip"
+First thing first, is to make sure you have python dowloaded to your PC. To do so,
```

Comment:
we want to target people who have never heard of Python before.  So I think an explanation about how to use the **online** jupyter notebook would be very helpful: https://mybinder.org/v2/gh/jupyterlab/jupyterlab-demo/try.jupyter.org?urlpath=lab